vast/integration/vast_integration_suite.yaml

```diff
@@ -402,6 +405,7 @@ tests:
       - command: show concepts
         transformation: jq '.[] | select(.concept.name == "net.app") | .concept.fields | map(split(".")[0]) | unique'
       - command: show --yaml models
+      - command: show --yaml schemas
```

Comment:
Maybe some test where we actually have data instead?

Reply:
I'll move the `--yaml` over to one of the other two uses.